include/polygeist/Passes/Passes.h

```diff
@@ -22,12 +22,21 @@ std::unique_ptr<Pass> createCPUifyPass(StringRef method = "");
 std::unique_ptr<Pass> createBarrierRemovalContinuation();
 std::unique_ptr<Pass> detectReductionPass();
 std::unique_ptr<Pass> createRemoveTrivialUsePass();
-std::unique_ptr<Pass> createParallelLowerPass();
+std::unique_ptr<Pass> createParallelLowerPass(bool wrapParallelOps = false);
+std::unique_ptr<Pass> createCudaRTLowerPass();
 std::unique_ptr<Pass>
 createConvertPolygeistToLLVMPass(const LowerToLLVMOptions &options,
-                                 bool useCStyleMemRef);
+                                 bool useCStyleMemRef, bool onlyGpuModules);
 std::unique_ptr<Pass> createConvertPolygeistToLLVMPass();
 std::unique_ptr<Pass> createForBreakToWhilePass();
+std::unique_ptr<Pass>
+createConvertParallelToGPUPass1(bool useOriginalThreadNums = false);
+std::unique_ptr<Pass> createConvertParallelToGPUPass2();
```

Comment:
Why is this called `createConvertParallelToGPUPass2` not `createConvertParallelToGPUPass`

Reply:
There are two of them: parallel-to-gpu1 and parallel-to-gpu2 because I want to use the upstream mlir gpu kernel outline pass (see driver.cc) and I could not find a nice way to bundle them together.